.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": ".NET 8 with .NET 6",
+    "image": "mcr.microsoft.com/dotnet/sdk:8.0",
+    "features": {
+		// https://github.com/devcontainers/features/tree/main/src/dotnet
```

Comment:
this comment links to the source of where I found which params to use for the snippet in lines 6-10